components-api/src/main/scala/pl/touk/nussknacker/engine/api/definition/ParameterValidator.scala

```diff
@@ -71,21 +87,26 @@ case object NotBlankParameterValidator extends ParameterValidator {
     nodeId
   )
 
-  private def isBlankStringLiteral(expression: String): Boolean =
-    blankStringLiteralPattern.matcher(expression.trim).matches()
+  private def isBlankStringLiteral(value: String): Boolean =
+    blankStringLiteralPattern.matcher(value.trim).matches()
```

Comment:
does the regex checking for whitespace only string make sense anymore when there are no surrounding "'" characters anymore and we trim the content?

Reply:
Excellent point, fixed that! :)